core/templates/dev/head/domain/exploration/QuestionListObjectFactory.js

```diff
@@ -35,6 +35,23 @@ oppia.factory('QuestionListObjectFactory', [function() {
     this._questions.push(question);
   };
 
+  QuestionList.prototype.removeQuestion = function(questionToDelete) {
+    var index = -1;
+    this._questions.some(function(question, idx) {
```

Comment:
I don't think I like the use of some() here -- it's a bit indirect. Why not just a regular for-loop with an explicit "break" statement?